generate.py

```diff
@@ -67,9 +77,37 @@ def generate(src="src", target="output", format="png", style="modern-blue"):
 
 
 def main():
-    generate("src", "output", "png",
-             "modern-blue")  # png only, svg & pdf must register and pay 15$.
-
+    styles = [
+        "default",
+        "rose",
+        "qsd",
+        "napkin",
+        "vs2010",
+        "patent",
+        "omegapple",
+        "modern-blue",
+        "earth",
+        "roundgreen",
+        "magazine"
+    ]
+    style_s = ", ".join(styles)
+
+    parser = OptionParser()
+    parser.add_option("-r", "--rebuild", dest="rebuild", default=False,
+                      help="Rebuild exist output image")
```

Comment:
I think this option don't need any additional argument, so is it better to set it action to `store_true` ?